pom.xml

```diff
@@ -20,6 +20,11 @@
   <name>GitHub API Plugin</name>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/GitHub+API+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.565</jenkins.version>
```

Comment:
Small bump greatly simplifies getting along with newer hpi plugin and JTH, as well as PCT checks. It is still old enough (and even before 1.580.x) not be an issue imo.